tools/cldr-apps/js/src/index.js

```diff
@@ -3,12 +3,17 @@ import "./style.css";
 
 import { createApp } from "vue";
 import About from "./views/About.vue";
+import Waiting from "./views/Waiting.vue";
 
```

Comment:
'Components names should always be multiwords like "AlertModal", "DropdownMenu" or "NavbarLogo"'
https://namingconvention.org/vuejs/components-naming.html
https://v3.vuejs.org/style-guide/#multi-word-component-names-essential

Reply:
Good point. 
- [ ] I'll pick some better names here.

Reply:
done changed to AboutPanel and WaitingPanel

Reply:
great!